hbase-server/src/main/java/org/apache/hadoop/hbase/util/DirectMemoryUtils.java

```diff
@@ -85,7 +85,8 @@ public static long getDirectMemorySize() {
     RuntimeMXBean runtimemxBean = ManagementFactory.getRuntimeMXBean();
     List<String> arguments = runtimemxBean.getInputArguments();
     long multiplier = 1; // for the byte case.
-    for (String s : arguments) {
+    for (int i = arguments.size() - 1; i >= 0; i--) {
```

Comment:
Does this method guarantee the order of the returned arguments?

Reply:
Thanks for reviewing Duo.

From what I see, the result is in order.
https://github.com/Tencent/TencentKona-8/blob/master/hotspot/src/share/vm/services/management.cpp#L545